cirq/contrib/routing/device.py

```diff
@@ -47,3 +47,30 @@ def gridqubits_to_graph_device(qubits: Iterable[cirq.GridQubit]):
 
 def _manhattan_distance(qubit1: cirq.GridQubit, qubit2: cirq.GridQubit) -> int:
     return abs(qubit1.row - qubit2.row) + abs(qubit1.col - qubit2.col)
+
+
+def nx_qubit_layout(graph: nx.Graph) \
+        -> Dict[Union[cirq.GridQubit, cirq.LineQubit], Tuple[int, int]]:
+    """Return a layout for a graph for nodes which are qubits.
+
+    This can be used in place of nx.spring_layout or other networkx layouts.
+    GridQubits are positioned according to their row/col. LineQubits are
+    positioned in a line.
+
+    >>> import cirq.contrib.routing as ccr
+    >>> import networkx as nx
+    >>> g = ccr.xmon_device_to_graph(cirq.google.Foxtail)
+    >>> pos = ccr.nx_qubit_layout(g)
+    >>> nx.draw_networkx(g, pos=pos)
+
+    """
+    pos = {
```

Comment:
Use inline type annotations now that we're python 3.6

`pos: Dict[...] = {}`

Reply:
nice